core/src/main/java/io/kestra/core/tasks/flows/Pause.java

```diff
@@ -76,6 +81,34 @@ public class Pause extends Sequential implements FlowableTask<VoidOutput> {
     @PluginProperty
     private Duration timeout;
 
+    @Valid
+    @PluginProperty
+    @Deprecated
+    private List<Task> tasks;
+
+
+    @Valid
+    @Deprecated
+    protected List<Task> errors;
```

Comment:
Errors should not be deprecated as it's part of the Flowable interface

Reply:
Will Pause still be a flowable later ? I don't see any reason, and errors is used for tasks in the flowable, no tasks, no errors ?

Reply:
Yes, pause will still be a flowable as it's executed by the executor.